tests/features/bootstrap/FeatureContext.php

```diff
@@ -83,32 +84,44 @@ public function iGotoAddaNewArticle()
      */
     public function iFillTheFollowingDetails(TableNode $table)
     {
-        throw new PendingException();
+        $page=$this->getSession()->getPage();
+        $tableArray = $table->getColumnsHash();
+        print_r($table->getRow(0));
+        $page->fillField("title",$tableArray[0]["title"]);
+        $page->fillField("summary", $tableArray[0]["summary"]);
+        $page->fillField("content", $tableArray[0]["content"]);
+        $page->fillField("publicationDate", $tableArray[0]["date"]);
     }
 
     /**
      * @Given I save the changes
      */
     public function iSaveTheChanges()
     {
-        throw new PendingException();
-    }
-
+    
+     echo "I save the changes";
+     $page=$this->getSession()->getPage();
+     $page->find('xpath', '//div/input[@name="saveChanges"]')->click();
+     }
+     
     /**
-     * @Then the article should be listed
+     * @Given I am on the New Article page
      */
-    public function theArticleShouldBeListed()
+    public function iAmOnTheNewArticlePage()
     {
-        throw new PendingException();
+        $this->getSession()->getPage()->find('xpath','//a[@href="admin.php?action=newArticle"]')->click();
```

Comment:
this will only work if you are already on the admin page, please use `visit()` to make it more universal